zavod/zavod/exporters/statistics.py

```diff
@@ -57,6 +75,12 @@ def observe(self, entity: Entity) -> None:
         self.schemata.add(entity.schema.name)
         for prop in entity.iterprops():
             self.qnames.add(prop.qname)
+        for prop_name, values in entity.properties.items():
+            # We add 1 instead of len(values) here because we want to count the number of entities that have this
+            # value set, not the number of values.
+            self.property_values_count[
+                SchemaProperty(entity.schema.name, prop_name)
+            ] += 1
```

Comment:
Now that we've switched to the (b) semantics, can I argue for `entities_with_prop` instead of `property_values`? Because we're asserting the number of entities which have some value for this prop. 

Happy to be rejected. It's pedantry but it's nice for things to be self-explanatory.